lms/djangoapps/student_dashboard/views.py

```diff
@@ -39,15 +39,15 @@ def get_recommended_courses(user):
     return recommended_courses
 
 
-def get_enrolled_past_courses(course_enrollments):
+def get_enrolled_past_courses(request, course_enrollments):
```

Comment:
We can avoid this newly added request parameter. You can take a look at 
`
from crum import get_current_request
request = get_current_request()
`